mfes/players/public/sunbird-plugins/renderer/org.sunbird.iframeEvent-1.0/renderer/plugin.js

```
@@ -0,0 +1,34 @@
+
+
+Plugin.extend({
+    initialize: function(){
+        console.info('Sunbird Telemetry get plugin is initialized..');
+        EventBus.addEventListener('telemetryEvent',this.sendTelemetry,this);
+    },
+    isPreviewInIframe: function(){
+        return (window.self != window.top) ? true : false;
+    },
+    sendTelemetry: function(evt) {
+        var instance = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)